src/lib/rest/HttpStatusCode.h

```diff
@@ -36,6 +36,7 @@ typedef enum HttpStatusCode
 {
   SccNone                   = 0,     // Undefined
   SccOk                     = 200,   // Success
+  // FIXME P10: SccFound 302 will be removed at the end once #787 gets implemented
```

Comment:
Yeah, I guess Sccfound will no longer be necessary,
as that information comes in a different way with the new implementation.

NTC